go.mod

```diff
@@ -6,39 +6,31 @@ go 1.14
 // should be updated in etc/bash_completion.d/rita
 // https://github.com/urfave/cli/blob/master/autocomplete/bash_autocomplete
 
+replace github.com/activecm/rita-bl => /home/logan/Development/ACM/rita-bl
```

Comment:
I used this while working on both projects since code needed to be changed in both places.
This needs to be replaced/ removed after the rita-bl PR goes through.